x-pack/plugin/ent-search/qa/rest/src/yamlRestTest/resources/rest-api-spec/test/entsearch/55_search_application_search.yml

```diff
@@ -52,7 +52,29 @@ setup:
 
   - do:
       search_application.put:
-        name: another-test-search-application
+        name: test-search-application-1
+        body:
+          indices: [ "test-search-index1", "test-search-index2" ]
+          analytics_collection_name: "test-analytics"
+          template:
+            script:
+              source:
+                query:
+                  term:
+                    "{{field_name}}": "{{field_value}}"
+              params:
+                field_name: field1
+                field_value: value1
+            dictionary:
+              additionalProperties: false
+              properties:
+                field_name:
+                  type: string
+                field_value:
+                  type: string
+  - do:
+      search_application.put:
+        name: test-search-application-with-list
```

Comment:
This search application was being called in a test, but wasn't actually being created.